content/ru/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -0,0 +1,349 @@
+
+Топологический домен X задается с помощью `topologyKey` — ключа для лейбла узла, который система использует для обозначения домена. Примеры см. в разделе [Типичные лейблы, аннотации и taint'ы](/docs/reference/labels-annotations-taints/).
+
+{{< note >}}
+Обработка правил совместного/раздельного существования подов требует значительных ресурсов и может значительно замедлить планирование в больших кластерах. Их не рекомендуется использовать в кластерах, число узлов в которых превышает несколько сотен.
+{{< /note >}}
+
+{{< note >}}
+Правила раздельного существования для подов требуют согласованности лейблов узлов. Другими словами, у каждого узла в кластере должен быть лейбл, относящий его к определенному ключу `topologyKey`. Отсутствие лейблов `topologyKey` у некоторых или всех узлов в кластере может привести к непредсказуемому поведению. 
+{{< /note >}}
+
+#### Типы правил совместного/раздельного существования подов
+
+По аналогии [c правилами для узлов](#node-affinity) существует два типа правил для подов:
+
+  * `requiredDuringSchedulingIgnoredDuringExecution`
+  * `preferredDuringSchedulingIgnoredDuringExecution`
+
+Например, с помощью правила совместного существования `requiredDuringSchedulingIgnoredDuringExecution` можно указать планировщику размещать поды, относящиеся к разным сервисам, в одной зоне облачного провайдера, поскольку они активно обмениваются данными друг с другом. 
+
+Аналогичным образом можно использовать правило раздельного существования `preferredDuringSchedulingIgnoredDuringExecution` для распределения подов по нескольким зонам облачного провайдера.
+
+Для задания правил совместного существования предназначено поле `affinity.podAffinity` в спецификации пода.
+
+Для задания правил раздельного существования предназначено поле `affinity.podAntiAffinity` в спецификации пода.
+
+#### Пример правил совместного/раздельного существования для пода {#an-example-of-a-pod-that-uses-pod-affinity}
+
+Рассмотрим следующую спецификацию пода:
+
+{{< codenew file="pods/pod-with-pod-affinity.yaml" >}}
+
+В примере задается одно правило совместного существования подов, и одно — раздельного. Для совместного правила используется жесткий тип `requiredDuringSchedulingIgnoredDuringExecution`, для раздельного — мягкий `preferredDuringSchedulingIgnoredDuringExecution`.
+
+Правило совместного существования гласит, что планировщик может разместить под на узел, только если тот находится в зоне с одним или более подами с лейблом `security=S1`. Точнее, планировщик должен разместить под на узле с лейблом `topology.kubernetes.io/zone=V`, если в этой зоне есть хотя бы один узел, на котором работают один или несколько подов с лейблом `security=S1`.
```

Comment:
This paragraph (and the next one) has a bit longer explanation now, so we need to update its translation as well.